Add instructions for partitioned cookies with the same name/domain/path (#9)
DCtheTall/CHIPS-spec: draft-cutler-httpbis-partitioned-cookies.md

      If the cookie-attribute-list does contain an attribute with an attribute-name of "Partitioned" and the secure-only-flag is false, abort these steps and ignore the cookie entirely.<br><br>
      Otherwise, set partition-key to cookie-partition-key defined in section 5.X.X.

## Partitioned Cookies with the Same Name/Domain/Path

> In order to prevent cross-partition leaks, we need to allow sites to set cookies with the same name, domain, and path as a cookie with another partition key.

Reviewer: Could we add the (brief) full rationale for this to the `Avoiding cross-partition leaks` section in Security Considerations?

Author: Done.

> In order to prevent cross-partition leaks, we need to allow sites to set cookies with the same name, domain, and path as a cookie with another partition key.

Reviewer: ```suggestion
In order to prevent cross-partition leaks, we need to allow sites to set cookies with the same name, domain, and path as long as they have different partition keys.
```

> In order to achieve this, we suggest the following edit to step 22 of 5.5 (Storage Model), note that steps b-d below are the same as the current spec.

Reviewer: ```suggestion
In order to achieve this, we suggest the following edit to step 22 of {{Section 5.5 (Storage Model) of RFC6265bis}}, note that steps b-d below are not changed.
```
